src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1347,14 +1348,27 @@ public long getRuinedTime() {
 	}
 
 	/**
-	 * Used by Dynmap-Towny to allow SiegeWar to set an occupying nation's colour.
+	 * Used by Dynmap-Towny to get the town's *local* map-colour.
 	 * 
 	 * @return String value of hex code or null.
 	 */
 	@Nullable 
+	public String getMapColorHexCode() {
```

Comment:
Pedantic, but this is missing an `@Override` annotation.

Reply:
Fixed